apps/src/music/appConfig.js

```diff
@@ -6,7 +6,7 @@ import {BlockMode} from './constants';
  * a default if an invalid value or no value is found.
  */
 export const getBlockMode = () => {
-  const defaultMode = BlockMode.TRACKS; // Current default is Tracks
+  const defaultMode = BlockMode.SIMPLE2; // Current default mode
```

Comment:
"Current default is Simple2"? 🙂

Reply:
My thinking was that the code was the source of truth and then we wouldn't need to change the comment each time we changed the value.

Reply:
Haha yeah that's true, we can probably just delete this comment then